views/site/login.php

```diff
@@ -29,7 +29,8 @@
         <?= $form->field($model, 'password')->passwordInput() ?>
 
         <?= $form->field($model, 'rememberMe')->checkbox([
-            'template' => "<div class=\"col-lg-offset-1 col-lg-3\">{input} {label}</div>\n<div class=\"col-lg-8\">{error}</div>",
+            // FIXME: NOPE not showing
+            'template' => "<div class=\"col-lg-offset-1 col-lg-3\">NOPE {input} {label}</div>\n<div class=\"col-lg-8\">{error}</div>",
```

Comment:
@simialbi any idea why `template` isn't working? Did the syntax change or is something broken?

Reply:
The offset syntax changed in bootstrap 4: `col-lg-offset-1` in bs3 is `offset-lg-1` in bs4:;
see https://getbootstrap.com/docs/4.2/layout/grid/#offsetting-columns

Reply:
https://github.com/yiisoft/yii2-bootstrap4/pull/100